index.html

```diff
@@ -2979,6 +2979,15 @@ <h2 id="notes">Notes</h2>
         </ul>
       </p>
 
+      <p id="relational-operator-note">
+        <b class="header">Underscore version</b>
```

Comment:
For consistency with the other API sections, I suggest naming this after the actual symbol.

```suggestion
        <b class="header">VERSION</b>
```